test/sql/cloud/glue/test_direct_keys_glue_no_endpoint_type.test

```diff
@@ -0,0 +1,97 @@
+# name: test/sql/cloud/glue/test_direct_keys_glue_no_endpoint_type.test
+# description: test integration with iceberg catalog read
+# group: [glue]
+
+require-env ICEBERG_AWS_REMOTE_AVAILABLE
+
+require-env AWS_ACCESS_KEY_ID
+
+require-env AWS_SECRET_ACCESS_KEY
+
+require avro
+
+require parquet
+
+require iceberg
+
+require httpfs
+
+require aws
+
+# test using keys directory
+statement ok
+CREATE SECRET s1 (
```

Comment:
For dumb reasons this secret may cause the attach to fail in CI, has to do with what permissions the account in aws secrets has. I would comment this out one for now

Reply:
but `test_direct_keys_glue.test` doesn't have this issue?